### PR TITLE
Changed concensus node for Prysm node and configurate eth node execut…

### PR DESCRIPTION
**Pull request by fagomezra**
…or in Kubernetes

### Title
Prysm node, and configure eth node executor

### What does this PR do?

- Change the lighthouse node for Prysm consensus node, and configure the eth node executor. #45 

### Steps to test
1.Execute the next commands in the Cluster Kubernetes to check the errors at nodes:
kubectl logs eth-executor-0 -n ethereum
kubectl logs prysm-beacon-0 -n ethereum

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Comment by xavier506** [comment]
extra space in this file name: `kubernetes/eth-executor--statefulset.yaml`

**Comment by fagomezra** [comment]
I change de name of kubernetes/eth-executor--statefulset.yaml wihtout extra space, and change the position of multilenguaje button.

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ const config = {
           },
           {
             type: 'localeDropdown',
-            position: 'left',
+            position: 'right',
```
Comment (kuronosec) [inline docs/docusaurus.config.js:90]: Do not mix PR that are for different topics.

**Review by kuronosec — APPROVED** [review]
LGTM

**Pull request merged**